Cargo.toml

```diff
@@ -77,7 +77,7 @@ custom_error = "1.9.2"
 decentralization = { path = "rs/decentralization" }
 trustworthy-node-metrics = { path = "rs/dre-canisters/trustworthy-node-metrics/src/trustworthy-node-metrics" }
 derive_builder = "0.20.0"
-derive_more = "0.99.18"
+derive_more = "*"
```

Comment:
Do we want to pick the latest compatible version always?

Reply:
You're right, let me nail it to version 1 (latest)